Cargo.toml

```diff
@@ -111,7 +111,7 @@ alloy = { version = "0.4.1", features = [
     "eips",
 ] }
 
-foundry-fork-db = "0.4"
+foundry-fork-db = "0.5"
 revm-primitives = "10.0"
 revm = "14.0"
```

Comment:
we also need to bump these

Reply:
thanks, this also fixes the `foundry-fork-db` example